templates/docs/base/forms.md

```diff
@@ -38,38 +38,80 @@ Note: The attribute `readonly` disables the input but it still retains a default
 
-<div class="embedded-example"><a href="/docs/examples/base/forms/aligned-checkboxes/" class="js-example">
-View example of checkboxes aligned to different headings
+<div class="embedded-example"><a href="/docs/examples/patterns/forms/checkbox-heading/" class="js-example">
+View example of the heading checkbox components
 </a></div>
 
+#### Deprecated base checkboxes
+
+<span class="p-label--deprecated">Deprecated</span>
+
+The base element styling for checkbox inputs (`<input type="checkbox">`) outside of the `.p-checkbox` component is now deprecated and will be removed in a future version of Vanilla framework.
+
+This applies also to all modifier classes on checkbox elements, such as `is-h1`, `is-h2`, `is-h3`, `is-h4`, `is-h5`, `is-muted-heading`, `is-muted-inline-heading`, `is-inline-label`, or `is-table-header`.
+
+Please refer to the [tick elements comparison example](/docs/examples/patterns/forms/tick-comparison) to find equivalent usage of new `.p-checkbox` component.
+
 ### Radio button
 
-Use radio buttons to select one or more options, our radio buttons can appear in four states: both selected, unselected and disabled.
+<span class="p-label--new">New</span>
+
+Use radio buttons to select one of the given set of options. To provide fully featured Vanilla style and behaviour of the radio button a specific markup structure is needed around the radio button input (see example below).
+
+To disable the radio component a `disabled` property needs to be added on respective `<input type='radio'>` element and `.is-disabled` class needs to be used on the component root `.p-radio` element.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/forms/radio/" class="js-example">
+View example of the radio components
+</a></div>
+
+#### Inline radio button
 
-<div class="embedded-example"><a href="/docs/examples/base/forms/radio-buttons/" class="js-example">
-View example of the base radio buttons
+To use radio component in padded containers (table cells, some list items) use the `.p-radio--inline` variant of the component to make sure the radio button and the label text are properly aligned with other inline text.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/forms/radio-inline/" class="js-example">
+View example of the inline radio components
 </a></div>
 
-<span class="p-label--updated">Updated</span>
+#### Heading radio button
 
-By default, radio buttons are vertically aligned to the baseline of text wrapped in a `label`, `h5`, `h6`, or `p` tag. If you need to align them to other elements, use one of the following classes:
-`is-h1`, `is-h2`, `is-h3`, `is-h4`, `is-h5`, `is-muted-heading`, `is-muted-inline-heading`, `is-inline-label`, or `is-table-header`.
+To use radio component in headings use the `.p-checkbox--heading` variant of the component to make sure the radio button and the label text are properly aligned.
```

Comment:
```suggestion
To style the radio label as a heading `.p-radio--heading` variant of the component to make sure the radio button and the label text are properly aligned.
```

Reply:
`--heading` variant doesn't style the label as heading, only make sure radio is aligned, so skipping this.